include/spatio_temporal_voxel_layer/spatio_temporal_voxel_grid.hpp

```diff
@@ -137,8 +137,8 @@ class SpatioTemporalVoxelGrid
   bool IsGridEmpty(void) const;
 
   // Get time information for clearing
-  double GetDecayShift(const double& time_delta);
-  double GetAcceleratedDecayShift(const double& time_delta, \
+  double GetBaseDurationToDecay(const double& time_delta);
+  double GetAcceleratedDecayModifier(const double& time_delta, \
```

Comment:
This is the acceleration factor, GetAccelerationDuration? or something along those lines

Reply:
`acceleration_factor` is already used as a parameter for this function--I'm not opposed to changing both if you have suggestions

Reply:
Acceleration duration?

Reply:
How about rename `GetAcceleratedDecayModifier -> GetAccelerationComponent` because it's not specifically intended to be used as a duration--it describes an influence on the resulting decay duration.

Reply:
The acceleration itself is the component, its not a modifier or anything, its what I would consider to be the acceleration term. GetFrustumAcceleration ?

Reply:
I agree, and that was the intent--I wanted to use the word "term" but that seemed a little too mathy. I like `GetFrustumAcceleration`

Reply:
:+1: do itt